builtin/credential/github/path_config.go

```diff
@@ -98,14 +98,14 @@ func (b *backend) Config(s logical.Storage) (*config, error) {
 	if err != nil {
 		return nil, err
 	}
+	if entry == nil {
```

Comment:
`organization` is required so we can't do anything with a default, *but* we already check for a blank organization later in `verifyCredentials`. So the behavior here is already correct.